trio/_core/_tests/test_run.py

```diff
@@ -249,46 +268,46 @@ async def test_current_time():
     assert t1 < t2
 
 
-async def test_current_time_with_mock_clock(mock_clock):
+async def test_current_time_with_mock_clock(mock_clock: _core.MockClock) -> None:
     start = mock_clock.current_time()
     assert mock_clock.current_time() == _core.current_time()
     assert mock_clock.current_time() == _core.current_time()
     mock_clock.jump(3.14)
     assert start + 3.14 == mock_clock.current_time() == _core.current_time()
 
 
-async def test_current_clock(mock_clock):
+async def test_current_clock(mock_clock: _core.MockClock) -> None:
     assert mock_clock is _core.current_clock()
 
 
-async def test_current_task():
+async def test_current_task() -> None:
     parent_task = _core.current_task()
 
-    async def child():
-        assert _core.current_task().parent_nursery.parent_task is parent_task
+    async def child() -> None:
+        assert not_none(_core.current_task().parent_nursery).parent_task is parent_task
 
     async with _core.open_nursery() as nursery:
         nursery.start_soon(child)
 
 
-async def test_root_task():
-    root = _core.current_root_task()
+async def test_root_task() -> None:
+    root = not_none(_core.current_root_task())
```

Comment:
Similar to above comment.

I'm not commenting on `test_current_task` I feel like getting a `None` there would be less an issue with the `current_task()` but idk, maybe that too?